cluster/ci/config/prow/plugins.yaml

```diff
@@ -700,22 +700,10 @@ config_updater:
       additional_namespaces:
       - ci-stg
 
-    ci-operator/templates/openshift/openshift-azure/build-base-image.yaml:
-      name: prow-job-build-base-image
-      additional_namespaces:
-      - ci-stg
-    ci-operator/templates/openshift/openshift-azure/build-node-image.yaml:
-      name: prow-job-build-node-image
-      additional_namespaces:
-      - ci-stg
     ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml:
       name: prow-job-cluster-launch-e2e-azure
       additional_namespaces:
       - ci-stg
-    ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-upgrade.yaml:
-      name: prow-job-cluster-launch-e2e-azure-upgrade
-      additional_namespaces:
-      - ci-stg
     ci-operator/templates/openshift/openshift-azure/rbac.yaml:
```

Comment:
Because you renamed file in https://github.com/openshift/release/pull/3155/files#diff-632f57953e70070a96b712e95f075aceR1 you need to change name too here

Reply:
Rollbacking the rename as it was not in the right place https://github.com/openshift/release/pull/3191